packages/toolpad-app/package.json

```diff
@@ -41,7 +41,7 @@
     }
   },
   "dependencies": {
-    "@auth/core": "0.24.0",
+    "@auth/core": "0.20.0",
```

Comment:
There's an issue in newer versions of `@auth/core` and I don't see any good solution proposed yet in https://github.com/nextauthjs/next-auth/issues?q=cannot+parse+action, will wait until they provide a fix or solution.

Reply:
Is there an exact issue?

Reply:
Nevermind it was an easy fix after all, I hadn't tried this for some reason: https://github.com/mui/mui-toolpad/pull/3056/commits/4445f9d00020f36a282cb1e29ca41bd7b53620c4